sdk/datamigration/azure-mgmt-datamigration/HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+2.3.0 (2019-07-12)
+++++++++++++++++++
+
+**Features**
+
+- Added operation group ServiceTasksOperations
+
```

Comment:
Please add the migration notes from v3 to v4:
https://github.com/Azure/azure-sdk-for-python/blob/master/tools/azure-sdk-tools/changelog_generics.rst#autorest-v3-to-v4-if-the-package-is-single-api

Reply:
done